.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,56 @@
+name: "Create release PR"
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: "Define release version (ex: v1, v2, v3)"
+        required: true
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check if remote branch exists
+        env:
+          BRANCH: releases/${{ github.event.inputs.release }}
+        run: |
+          echo "##[set-output name=exists;]$(echo $(if [[ -z $(git ls-remote --heads origin ${BRANCH}) ]]; then echo false; else echo true; fi;))"
+        id: extract-branch-status
+      # these two only need to occur if the branch exists
+      - name: Checkout proper branch
+        if: ${{ steps.extract-branch-status.outputs.exists == 'true' }}
+        env:
+          BRANCH: releases/${{ github.event.inputs.release }}
+        run: git checkout ${BRANCH}
+      - name: Reset promotion branch
+        if: ${{ steps.extract-branch-status.outputs.exists == 'true' }}
+        run: |
+          git fetch origin master:master
```

Comment:
these two lines need to change from master to main for this repo

Reply:
changed